### PR TITLE
Feature/date range picker testing

### PR DESCRIPTION
**Pull request by musangowope**
## Proposed changes

- Updated the yarn lock file
- Added button type to Mobile Side Menu
- Added one story for date range picker
- Added test for date range picker

...

## Types of changes

What type of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] New feature
- [ ] Bugfix
- [x] Quality of Life

## Checklist

_Put an `x` in the boxes that apply._

- [x] Linter passes (`yarn lint`)
- [ ] Unit Test pass (`yarn jest`)
- [x] I have added at least two useful tests for new component (if applicable)
- [x] I have added at least one Storybook story (if applicable)

## Relevant Issues
resolves https://github.com/bonlineza/b-lib/issues/48

### CODE REVIEW

File: src/components/DateRangePicker/index.test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    wrapper = setup();
+  });
+
+  it('without errors', () => {
+    expect(wrapper.length).toBe(1);
+  });
+
+  it('startDatePlaceholderText prop`s default value is `Start Date`', () => {
+    const { startDatePlaceholderText } = wrapper.props();
+    expect(startDatePlaceholderText).toBe('Start Date');
```
Comment (sscholle) [inline src/components/DateRangePicker/index.test.js]: i think youre missing the point of the test - don't test implementation

File: src/components/DateRangePicker/index.js [hunk 2]
```diff
@@ -24,7 +34,7 @@ const DateRangePicker = ({
     }
   }, [init]);
```
Comment (sscholle) [inline src/components/DateRangePicker/index.js]: i'm not sure if init is still being used

File: src/components/DateRangePicker/index.test.js (hunk 1 above)
Comment (sscholle) [reply src/components/DateRangePicker/index.test.js]: 1. most importantly: see if you cant enter a date value into the field using 'simulate.change()' and see if the onDataChanged triggers
2. secondly: check that passed in props are represented and reflected in the component

File: src/components/DateRangePicker/index.js (hunk 2 above)
Comment (musangowope) [reply src/components/DateRangePicker/index.js]: refactored

File: src/components/DateRangePicker/index.test.js (hunk 1 above)
Comment (musangowope) [reply src/components/DateRangePicker/index.test.js]: done

**Pull request merged**